dnf5.spec

```diff
@@ -376,6 +376,7 @@ Python 3 bindings for the libdnf library.
 
 %files -n python3-libdnf5
 %{python3_sitearch}/libdnf5
+%{python3_sitearch}/libdnf5-*.dist-info
```

Comment:
I suppose this could also use the global project major/minor/patch vars to avoid the wildcard- can update that to do so if desired.